libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -73,6 +86,11 @@ class AP_InertialSensor_Backend
     // notify of a fifo reset
     void notify_fifo_reset(void);
     
+    float get_pitch_angle(void);
+    float get_yaw_angle(void);
```

Comment:
These function names are misleading. They are FFT specific, so the name should say that somewhere.

Reply:
ok